cypress/integration/onsite/onsite_group_landing_page_spec.js

```diff
@@ -36,16 +36,16 @@ describe('Given I navigate to /onsite/group Page:', function() {
       .should('have.length', onsiteGroupList.length);
   });
 
-  it('Onsite Group card for Financial Peace should be last', function() {
+  it('Onsite Group card for Financial Peace should be first', function() {
     cy.get('.col-md-4').as('onsiteGroupCards')
-      .last()
+      .first()
       .find('a')
       .should('have.attr', 'href', '/groups/onsite/financial-peace/uptown');
   });
 
-  [0, 1, 2].forEach((index) => {    
+  [1,2,3,].forEach((index) => {    
     it(`Onsite card #${index} should have a Title`, function() {
-      let title = onsiteGroupList[index].title.text;
+      let title = onsiteGroupList[index + 8].title.text;
```

Comment:
Reporting that we're testing card at index 1 when we're actually testing the card at index 9 is very misleading. Can you change the array from 1-3 to 9-11 if those are the indexes of the cards that should be tested?